catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/session/session_cubit.dart

```diff
@@ -174,11 +158,14 @@ final class SessionCubit extends Cubit<SessionState>
     );
   }
 
-  SessionState _createMockedSessionState() {
+  Future<SessionState> _createMockedSessionState() async {
```

Comment:
100% agree. I've left TODO in SessionState because of this reason, [here](https://github.com/input-output-hk/catalyst-voices/pull/1391/files#diff-c23482754e16fd1e3319ee4d1d83fff384a310c42f05e04ac656c9482722e644R57). `SessionState` should not have whole `Account` for this reason and fact of exposing Keychain this way.

Real account is only accessible from `UserService`. This one is not changed here